### ISSUE TITLE
PeoplePicker: Cannot delete value from the control on Edge with Narrator ON

### ISSUE THREAD

**Issue by ssingh28** [OfficeDev/office-ui-fabric-react#5287]
This issue can be observed even on the MS Fabric page https://developer.microsoft.com/en-us/fabric#/components/peoplepicker    on Edge with Narrator ON

The only way people picker works is if the control comes via mouse rather than keyboard. Also once the item is chosen if backspace/delete is clicked Narrator reads out "unavailable". This same thing is read out when someone tries to type a name is the control. Attached is the video of the same.

**Comment by natalieethell** [comment]
Hey @ssingh28 thanks for submitting this issue! I'm currently not able to repro this behavior. Here's what I'm seeing in Edge with Narrator ON:

![peoplepicker in narrator](https://user-images.githubusercontent.com/14134000/43980419-e1951ad6-9ca2-11e8-9707-b0d8c8be4097.gif)

I'm able to select items and delete them using the keyboard.

Would you mind checking the behavior again to see if it's what you're expecting? Thanks!

**Comment by JasonGore** [comment]
Covered by bug 13 of #6259. Closing as duplicate.
